tests/holdings_maintenance/support.py

```
@@ -0,0 +1,649 @@
+        stdout: What the tool printed -- its logger output and its reports.
+        stderr: Everything else the process wrote: tracebacks, and any warning an
+            imported library chose to emit.
+        output: stdout followed by stderr, for assertions that do not care which
+            stream a message arrived on. The two are separated by a newline, so no
+            match can span them: a tool whose last stdout line has no terminator
+            would otherwise fuse it with stderr's first line and could satisfy --
+            or wrongly break -- a substring assertion that neither stream alone
+            does.
+    """
+
+    def __init__(self, argv, returncode, stdout, stderr):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)